recipes-kernel/linux/linux-tegra_4.9.bb

```diff
@@ -9,6 +9,7 @@ require recipes-kernel/linux/linux-yocto.inc
 
 DEPENDS:remove = "kern-tools-native"
 DEPENDS:append = " kern-tools-tegra-native"
+DEPENDS:append = " ${@bb.utils.contains('DISTRO_FEATURES', 'wifi', 'wireless-regdb-native', '', d)}"
```

Comment:
To make this easier for users to override, could you add a new variable to control this, with the default being based on `wifi` being in DISTRO_FEATURES? Something like
```
KERNEL_INTERNAL_WIRELESS_REGDB ?= "${@bb.utils.contains('DISTRO_FEATURES', 'wifi', '1', '0', '', d)}"
```
and then base all of the conditionals on that variable, as in
```
DEPENDS:append = "@' wireless-regdb-native' if bb.utils.to_boolean(d.getVar('KERNEL_INTERNAL_WIRELESS_REGDB')) else ''"
```
That will make it easier for a user to switch to using `crda` if they prefer.

Reply:
Agree 100%

Reply:
Will push the changes